pkg/sign/signer.go

```diff
@@ -87,7 +95,7 @@ func Bundle(content Content, keypair Keypair, opts BundleOptions) (*protobundle.
 			Bytes: pubKeyBytes,
 		})
 
-		// TODO: do verification of Fulcio certificate
+		verifierOptions = append(verifierOptions, verify.WithSignedCertificateTimestamps(1))
```

Comment:
The Fulcio instance isn't guaranteed to be configured to write to a CT log. We should condition this on the SCT extension (`asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 11129, 2, 4, 2}`, from https://github.com/sigstore/fulcio/blob/e813f854fe72d22fa9a9af8961dcda234d8d8f6f/pkg/ca/baseca/baseca.go#L37C22-L37C77) is set in the certificate. Marshal `pubKeyBytes` to a `x509.Certificate`, and:

```
for _, ext := range cert.Extensions {
    if reflect.DeepEqual(ext.Id, asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 11129, 2, 4, 2}) {
        verifierOptions = append(verifierOptions, verify.WithSignedCertificateTimestamps(1))
        break
    }
}
```